src/conductor.ts

```diff
@@ -162,13 +164,13 @@ export class Conductor {
 
   // this function will auto-generate an `installed_app_id` and
   // `dna.nick` for you, to allow simplicity
-  installHapp = async (agentHapp: T.DnaSrc[], agentPubKey?: AgentPubKey): Promise<T.InstalledHapp> => {
+  installHapp = async (agentHapp: T.InstallHapp, agentPubKey?: AgentPubKey): Promise<T.InstalledHapp> => {
```

Comment:
This introduces a breaking change to all tryorama tests that requires using an object instead of the just the DNA source for installHapp.  And part of the idea behind the current tryorama approach was to reduce configuration and just let tryorama set things for you.  But I understand the need for being able to set the hAppId.  I would like for us to have a non-breaking change way to allow for this.  Also in #71 you can do this if you use the bundled form for installation.

Reply:
Great, let's just manage this in the bundles then!  The only reason I was (begrudgingly) ok with introducing the breaking change was because there was need in the integration setup and tests.

I'll go ahead and remove the `installed_app_id` option then, and we can just move along with the app port config. :+1:

Reply:
+1